letsencrypt-apache/letsencrypt_apache/constants.py

```diff
@@ -27,3 +27,15 @@
 REWRITE_HTTPS_ARGS = [
     "^", "https://%{SERVER_NAME}%{REQUEST_URI}", "[L,QSA,R=permanent]"]
 """Apache rewrite rule arguments used for redirections to https vhost"""
+
+
+HSTS_ARGS = ["always", "set", "Strict-Transport-Security",
+    "\"max-age=31536000; includeSubDomains\""]
```

Comment:
We discussed it on `#letsencrypt-dev`: We'll create an API wrapper on top of augeas for a more versatile http header placements (for instance: adaptive HSTS max-age / HPKP). In the mean time we'll keep the simple http-header placement method that is contained in the PR.